src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusSerializer.cs

```diff
@@ -173,11 +173,11 @@ public static int WriteLabelKey(byte[] buffer, int cursor, string value)
     }
 
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    public static int WriteLabelValue(byte[] buffer, int cursor, string value)
+    public static int WriteLabelValue(byte[] buffer, int cursor, string? value)
     {
         Debug.Assert(value != null, $"{nameof(value)} should not be null.");
```

Comment:
Can line 178 be changed to use a `Guard` method?

Reply:
I reverted this change. This method does not really support `value` being `null` so it shouldn't be decorated as `string?`.